WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -33,6 +33,14 @@ class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	 */
 	const ERROR_MSG = '%s by a theme/plugin should start with the theme/plugin prefix. Found: "%s".';
 
+	/**
+	 * Minimal number of characters the prefix needs in order to be valid.
+	 *
+	 * @see Issue #1733
```

Comment:
Let's change this issue number into a standard URL, so that it becomes clickable in some editors, and doesn't presume that the person reading this snippet of code knows where the correct repo lives.